### PR TITLE
rofi-calendar: navigation lines to increment and decrement month

### PR DESCRIPTION
**Pull request by Aster89**
Changes:
  1) Added two navigation lines (at the bottom) to increment/decrement by
    one month (the previous selection is kept active)
  2) Added option to show previous month on left click (otherwise left and
    middle click show current month, right click shows next month)

### CODE REVIEW

**Comment by Aster89** [comment]
I've also used and the injected properties, and checked that they work.

**Review by kb100 — DISMISSED** [review]

File: rofi-calendar/rofi-calendar [hunk 1]
```diff
@@ -1,40 +1,135 @@
...
+      selected="$(echo $month_page \
+        | rofi \
+            -dmenu \
+            -markup-rows \
+            -font $FONT \
+            -m -3 \
+            -lines $(echo $month_page | wc -l) \
+            -width -25 \
+            -theme-str $THEME_STR \
+            -theme "$ROFI_CONFIG" \
```
[the remark this answers is not in the document]
Comment (Aster89) [reply rofi-calendar/rofi-calendar]: By _outside configuration_ you're referring to the `-theme-str` and `-theme` options, not the preceding two, right?

[the remark this answers is not in the document]
Comment (kb100) [reply rofi-calendar/rofi-calendar]: Apologies for not being clear the first time. The problem I have is that I have no rofi config, so $ROFI_CONFIG is a file that I consider outside configuration preventing the block from working for me. Somehow the case where no rofi config is specified needs to be handled in a way that doesn't crash. Theoretically, you could inline default rofi config options and use them if $ROFI_CONFIG file does not exist, that would be okay. I just don't want the block totally dependent on some file I don't have.

File: rofi-calendar/rofi-calendar [hunk 2]
```diff
@@ -1,40 +1,135 @@
 #! /bin/sh
```
[the remark this answers is not in the document]
Comment (Aster89) [reply rofi-calendar/rofi-calendar]: I've handled that case by using `/dev/null` as default config file. It seems it works.

File: rofi-calendar/rofi-calendar (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aster89) [reply rofi-calendar/rofi-calendar]: I've handled that case by using `/dev/null` as default config file. It seems it works.

**Comment by Aster89** [comment]
@kb100 , I think I've addressed the problems you pointed out.

**Comment by kb100** [comment]
Which `cal` are you using? The cal on my system (debian testing) does not have `--color=always` or `--monday`.

**Comment by Aster89** [comment]
The `--color=always` option has been there well before I implemented the two navigation lines. Concerning the `--monday` option, I've forgot to remove it, but anyway I can't understand how it could affect anything...

I'll check asap.

**Comment by Aster89** [comment]
@da-edra , can you give a look at this?

**Comment by Aster89** [comment]
@da-edra , @jolange any of you can give a look at this? It's stagnating for quite a long while :/

**Comment by da-edra** [comment]
Hi @Aster89 sorry for the late reply, looking at it now. 😉

**Review by da-edra — APPROVED** [review]
This looks **awesome**! 

Thanks a lot! 🤗

**Pull request merged**